cypress.config.js

```diff
@@ -10,10 +10,17 @@ module.exports = defineConfig({
           const email = faker.internet.email();
           const randomNumber = Math.floor(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + randomNumber,
+            username: faker.person.firstName() + randomNumber,
             email: email.toLowerCase(),
             password: '12345Qwert!'
           };
+        },
+        generateArticle() {
+          return {
+            title: faker.word.verb(),
+            description: faker.word.words(),
+            body: faker.lorem.paragraph(2)
```

Comment:
As a suggestion, you might want to generate a more realistic article body, maybe using `faker.lorem.paragraphs()` to get multiple paragraphs.